Taarafo.Core.Tests.Unit/Services/Foundations/Comments/CommentServiceTests.Logic.RetrieveById.cs

```diff
@@ -0,0 +1,49 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using System;
+using System.Threading.Tasks;
+using Taarafo.Core.Models.Comments;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Comments
+{
+    public partial class CommentServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveCommentByIdAsync()
+        {
+            // given
+            Guid randomCommentId = Guid.NewGuid();
+            Comment randomComment = CreateRandomComment();
+            randomComment.Id = randomCommentId;
```

Comment:
I am still wondering if we need this. We are generating a random comment, then can we use the random comment's id as the inputCommentId?

Reply:
@ShriHumrudha I agree - this is not needed.
@cjdutoit you don't need to reassign the id. all we care about is that when a particular random id is passed an object comes back from the broker. that's all.

Reply:
@ShriHumrudha, I have updated this